src/components/errorbars/plot.js

```diff
@@ -22,8 +22,8 @@ module.exports = function plot(traces, plotinfo) {
 
     traces.each(function(d) {
         var trace = d[0].trace,
-            xObj = trace.error_x,
-            yObj = trace.error_y;
+            xObj = trace.error_x || {},
+            yObj = trace.error_y || {};
```

Comment:
Ahh. I guess that cleaner than making `scatter.plot` takes an `opts` flag for error bars. 

That said, can you add a comment about why we that fallback?